packages/server/migrations/20230119175045_add_parsed_data_blod_for_uploads.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
Shouldn't `blod` in the title be `blob`? (It's possible `blod` is a term of art I'm not familiar with?

Reply:
whoops, I typo'd that. Will fix it now!